src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -557,6 +557,13 @@ static smt_termt convert_expr_to_smt(const shift_exprt &shift)
       convert_expr_to_smt(left_shift->op0()),
       convert_expr_to_smt(left_shift->op1()));
   }
+  else if(
```

Comment:
:bulb: If you get rid of the unneeded `else` key word from here, this if condition will all fit on one line of code.